mix.exs

```diff
@@ -6,7 +6,11 @@ defmodule Avalanche.MixProject do
 
   @version_file Path.join(__DIR__, ".version")
   @external_resource @version_file
-  @version (case Regex.run(~r/^([\d\.\w-]+)/, File.read!(@version_file), capture: :all_but_first) do
+  @version_file_content (case File.read(@version_file) do
+                           {:ok, content} -> content
+                           _ -> "0.0.0"
+                         end)
```

Comment:
Is this better than creating a `.version` file with `0.0.0` as a placeholder? Less clutter in the repo and root folder. So probably.

Reply:
I think that mix doesn't copy the `.`files when moving the deps inside the deps folder. The file is good in my opinion, but mix doesn't copy it over causing it to cry

Reply:
maybe we create file without the `.` -> `version` with `0.0.0` inside. Thoughts?

Reply:
actually, I see `.`files in other deps. Is it possible we are not pushing that file over to hex?